docs/data/migration/migration-pickers-v5/migration-pickers-v5.md

```diff
@@ -19,6 +19,24 @@ Since v6 is a major release, it contains some changes that affect the public API
 These changes were done for consistency, improve stability and make room for new features.
 Below are described the steps you need to make to migrate from v5 to v6.
 
+### Drop `clock` in desktop mode
+
+We have dropped the clock picker behavior in desktop mode for `DateTimePicker` and `TimePicker` components.
+This is the first step towards moving to a [better implementation](https://github.com/mui/mui-x/issues/4483).
+The behavior on mobile mode is still the same.
+If you were relying on Clock Picker in desktop mode for tests—make sure to check [testing caveats](/x/react-date-pickers/getting-started/#testing-caveats) to choose the best replacement for it.
+
+### Drop direct `@date-io` adapters support
+
+In order to support new Date and Time pickers with [fields components](/x/react-date-pickers/fields/) we had to augment `@date-io` adapters.
```

Comment:
In v5, it was possible to import adapters either from `@date-io` or `@mui/x-date-pickers` which were the same.
In v6, the adapters are extended by `@mui/x-date-pickers` to support field components. 
Which means adapters can not be imported from `@date-io` anymore. They might be imported from `@mui/x-date-pickers` or `@mui/x-date-pickers-pro`. Otherwise, some methods will be missing.

Reply:
Updated with some tweaks to the whole paragraph. 👌